packages/harmony/src/components/artwork/Artwork.tsx

```diff
@@ -0,0 +1,61 @@
+import { ComponentProps, useState } from 'react'
+
+import { useTheme } from '@emotion/react'
+
+import { Box, BoxProps } from 'components/layout'
+import { Skeleton } from 'components/skeleton'
+
+export type ArtworkProps = { isLoading?: boolean } & Pick<
+  ComponentProps<'img'>,
+  'src'
+> &
+  BoxProps
+/**
+ * The artwork component displays the track cover art and appears in several
+ * locations such as track tiles, track and playlist, pages,
+ * and the sidebar. It can have interactive elements on hover.
+ * This component enhances the listening experience and helps users quickly
+ * identify their favorite tracks.
+ */
+export const Artwork = (props: ArtworkProps) => {
+  const { isLoading: isLoadingProp, src, ...other } = props
+  const [isLoadingState, setIsLoading] = useState(isLoadingProp ?? true)
+  const isLoading = isLoadingProp ?? isLoadingState
+  const { motion } = useTheme()
+
+  return (
+    <Box borderRadius='s' border='default' {...other}>
+      {isLoading ? (
+        <Skeleton
+          borderRadius='s'
+          h='100%'
+          w='100%'
+          css={{ zIndex: 1, position: 'absolute' }}
```

Comment:
from what i remember, zIndex is always relative to the parent, so i think we should be good? since this is an internal implementation detail and not part of the actual app layering system, i just went with "1", thoughts?